app/utils/logger.js

```diff
@@ -27,7 +28,7 @@ logConfig.streams.push({
 
 const parent = bunyan.createLogger(logConfig);
 const loggerCollector = [parent];
-process.on('SIGUSR2', () => {
+process.once('SIGUSR2', () => {
```

Comment:
are you sure the once is correct approach?

Reply:
Sure. In nodemon README [it's described](https://github.com/remy/nodemon#controlling-shutdown-of-your-script)